README.md

```diff
@@ -16,16 +16,19 @@ and letting the container prepare that source code for execution. The base
 S2I container images contain the language runtime and build tools needed for
 building and running the source code.
 
+To install `s2i` use [Openshift Tools Installer]
 This Action will install [the latest](https://github.com/openshift/source-to-image/releases/tag/v1.3.1) version of S2I.
 
+To install any specific version of `s2i` use [Openshift Tools Installer](https://github.com/marketplace/actions/openshift-client-installer).
+
 **NOTE:**
 `s2i-build` only works on Linux platforms, because it relies on the Docker daemon.<br>
 If you are using GitHub's Ubuntu runners, the Docker daemon will already be available.
 Otherwise, you can use [Docker Setup Buildx](https://github.com/marketplace/actions/docker-setup-buildx) to set up and start the Docker daemon.
 
-Also see [buildah-build](https://github.com/redhat-actions/buildah-build) for more configurable method of building images, from scratch or from a Dockerfile.
+Also see [Buildah Build](https://github.com/marketplace/actions/buildah-build) for more configurable method of building images, from scratch or from a Dockerfile.
```

Comment:
let us bold these, and write them in their slug forms.

```
also see [**buildah-build**]
```

Reply:
Okay